packages/bento-design-system/src/Table/Table.tsx

```diff
@@ -593,6 +618,60 @@ function ColumnHeader<D extends Record<string, unknown>>({
   );
 }
 
+function ColumnFooter<D extends Record<string, unknown>>({
+  column,
+  style,
+  lastLeftSticky,
+  stickyFooters,
+  sticky,
+  first,
+  last,
+}: {
+  column: ColumnInstance<D> | HeaderGroup<D>;
+  style: CSSProperties;
+  lastLeftSticky: boolean;
+  stickyFooters: boolean;
+  sticky: boolean;
+  first: boolean;
+  last: boolean;
+}) {
+  const config = useBentoConfig().table;
+
+  return (
+    <Box
+      className={[lastLeftSticky && lastLeftStickyColumn, stickyFooters && stickyColumnFooter]}
+      style={{
+        ...style,
+        zIndex: sticky ? zIndexes.leftStickyHeader : zIndexes.header,
+      }}
+    >
+      <Box
+        className={columnFooter}
+        background={config.footerBackgroundColor}
+        color={config.footerForegroundColor}
+        {...column.getFooterProps()}
+        textAlign={column.align}
+        {...config.padding.footer}
+      >
+        <Box
+          paddingLeft={first ? config.boundaryPadding : undefined}
+          paddingRight={last ? config.boundaryPadding : undefined}
+        >
+          {column.Footer && (
+            <Columns space={8} alignY="center" align={column.align}>
+              {column.Footer ? (
```

Comment:
isn't this already checked by the `&&` at line 660? 
Also, do we want the wrapping Boxes if `column.Footer` is null?